www/src/css/lib/vanilla-overrides.scss

```diff
@@ -5,4 +5,50 @@
 // If any styles from vanilla framework need to be overridden, corresponding
 // SCSS rules should be put in this file (not mixed with other styles).
 
+// scss/modules/_header.scss
+.banner .nav-primary ul li:hover {
+  background-color:#f7f7f7;
+}
 
+.banner .nav-primary ul li a,
+.banner .nav-primary ul li a:link,
+.banner .nav-primary ul li a:visited {
+  background-color:transparent;
+  font-size:initial;
+  border-left:0;
+  padding:14px 12px 8px 12px;
```

Comment:
These were set to achieve a specific size and position in relation to the logo for the most part.

Without these overrides the nav looks roughly like this:

![image](https://cloud.githubusercontent.com/assets/45121/16077782/232c755a-3325-11e6-9ed2-f4d9a1c2f411.png)

Versus with the overrides:

![image](https://cloud.githubusercontent.com/assets/45121/16077793/2e26fea8-3325-11e6-9a0a-80974e6ea3b4.png)

Maybe @jamiedawsonyoung or @kwm14 can chip in here?